### PR TITLE
freexl [skip appveyor]

### PR DESCRIPTION
**Pull request by ocefpaf**
Another optional dependency for `gdal`.

### CODE REVIEW

**Comment by ocefpaf** [comment]
Ping @jakirkham can you take a quick look and merge this? (It is in the right directory :wink:)

File: recipes/freexl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    url: http://www.gaia-gis.it/gaia-sins/freexl-{{ version }}.tar.gz
+    md5: 9954640e5fed76a5d9deb9b02b0169a0
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+    run:
```
Comment (jakirkham) [inline recipes/freexl/meta.yaml]: Should we just drop these?

File: recipes/freexl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    commands:
+        - test -f ${PREFIX}/lib/libfreexl.a  # [unix]
+        - test -f ${PREFIX}/lib/libfreexl.so  # [linux]
+        - test -f ${PREFIX}/lib/libfreexl.dylib  # [osx]
+        - conda inspect linkages -n _test freexl  # [linux]
+
+about:
+    home: https://www.gaia-gis.it/fossil/freexl/index
+    license: LGPL-2.1
+    summary: Extract valid data from within an Excel
```
Comment (jakirkham) [inline recipes/freexl/meta.yaml]: Nifty. :smile:

**Comment by jakirkham** [comment]
One minor comment. Otherwise LGTM.

File: recipes/freexl/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline recipes/freexl/meta.yaml]: I thought the linter would complain about its absence. Removed!

Comment (jakirkham) [inline recipes/freexl/meta.yaml]: Ah, yeah, I've left it out before and it's been ok.

**Pull request merged**